src/routes/welcome/components/index.tsx

```diff
@@ -44,7 +44,7 @@ const CardsCol = styled.div`
   width: 50%;
 `
 const StyledButton = styled(Button)`
-  margin-top: auto;
+  margin-top: auto !important;
```

Comment:
why?

Reply:
Somehow it stopped working when I touched some seemingly unrelated button CSS.